mythguy1226/80sgame
# added fill and outline panel assets

Added Two UI Panel assets:
* Filled
* Outline

To Test:
* Open the achievements menu and check out the changes using the above assets to see if it looks good.

Relevant Info:
* To use these assets, set a panel's source image to the filled one. Then, add a child panel and set its source image to the outline if wanted.
* To adjust size of corner pixels, change the Pixels Per Unit Multiplier setting under the Image component of the panel.
* For outlines to appear correct, make sure both panels use the same multiplier.

Images:
![image](https://github.com/user-attachments/assets/5699e4b9-1b7e-4827-831d-46ea0f532c4d)
![image](https://github.com/user-attachments/assets/ebac3f7e-6a97-4241-b8c3-4e0bf0ff0718)
![image](https://github.com/user-attachments/assets/5578d56d-3dc9-4965-871a-2109dc81e233)

Issue: https://bat-bots.atlassian.net/jira/software/projects/BB/boards/1?selectedIssue=BB-310

Due Sprint 6
